config/settings/base.py

```diff
@@ -525,6 +525,9 @@
     "VARFISH_CADA_REST_API_URL", "https://cada.gene-talk.de/api/process"
 )
 
+# Enable Face prioritization.
+VARFISH_ENABLE_FACE = env.bool("VARFISH_ENABLE_FACE", default=True)
```

Comment:
`FACE` should be renamed to `GM` or `GESTALTMATCHER`.

Reply:
done